src/inmanta/util/__init__.py

```diff
@@ -888,37 +887,37 @@ def remove_comment_part(to_clean: str) -> str:
     return drop_comment
 
 
+"""
+A CanonicalRequirement is a packaging.requirements.Requirement except that the name of this Requirement is canonicalized, which
+allows us to compare requirements without dealing afterwards with the format of these requirements.
```

Comment:
```suggestion
allows us to compare names without dealing afterwards with the format of these requirements.
```

Requirements can be compared regardless because `packaging` compares canonical forms.